### PR TITLE
feat: added the cards component

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Cards Component:
- Cards with Badges
image

- Cards with Dismiss
image

- Cards with Text Overlay
image

- Cards - Text Only: horizontal and vertical
image

- Cards - Horizontal/Vertical: with and without images
image

- Cards with Shadow
image

### CODE REVIEW

File: docs/cards.html [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+                            <li>Grey: <code class="class-highlight">grey-badge</code></li>
+                            <li>Green: <code class="class-highlight">green-badge</code></li>
+                            </ul>
+                    </p>
+                </div>
+                <div class="component-showcase cards-w-badges">
+                    <div class="card-container">
+                        <div class="card-child card-header">
+                            <img class="card-img"
+                                src="https://m.media-amazon.com/images/I/71qLvcSA2lL._AC_UL480_FMwebp_QL65_.jpg" alt="">
```
Comment (anshulraheja) [inline docs/cards.html]: add something in alt tag

**Review by anshulraheja — APPROVED** [review]

File: docs/cards.html [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+                            <p class="card-desc  p-sm">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempora
+                                quae sapiente cum. Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus nemo
+                                atque alias velit earum voluptate magni. Eos deserunt cum voluptatibus laborum.
+                                Corrupti, vel nobis incidunt culpa sequi voluptatum totam dicta!</p>
+                        </div>
+                        <div class="card-child card-footer">
+                            <div class="actions-wrapper">
+                                <button class="action-btn p-sm">Read More</button>
+                                <button class="action-btn p-sm">Bookmark</button>
+                            </div>
```
Comment (nisargpawade) [inline docs/cards.html:221]: the buttons and the badges look similar, try increasing padding or font-size so they will look more like buttons

**Review by nisargpawade — APPROVED** [review]
everything else looks fine

**Review by kushalsourav — APPROVED** [review]
Everything looks good

File: docs/cards.html (hunk 1 above)
Comment (SinchanaMS) [reply docs/cards.html]: Oh! I missed it. Thanks, Anshul!

File: docs/cards.html (hunk 2 above)
Comment (SinchanaMS) [reply docs/cards.html:221]: Thank you, will work on it.

**Comment by SinchanaMS** [comment]
Made the changes as per PR review. Merging to dev.

**Pull request merged**